src/task.ts

```diff
@@ -0,0 +1,38 @@
+import { Manager } from '@twilio/flex-webchat-ui';
+import { Channel } from 'twilio-chat/lib/channel';
+
+export type TaskState = {
+  currentSid?: string;
+};
+
+const initialState: TaskState = {};
+
+const SET_CURRENT_TASK = 'setCurrentTask';
+
+type SetCurrentTaskAction = {
+  type: typeof SET_CURRENT_TASK;
+  newSid: string | undefined;
+};
+
+const setCurrentTaskFromChannel = (channel: Channel): SetCurrentTaskAction => {
+  const attributes: { taskSid: string } = channel.attributes as any;
+  console.log('Channel State Changed', attributes, channel);
```

Comment:
We should also remove this `console.log` because it will log to the client's website.